app/Models/Wiki/Page.php

```diff
@@ -227,6 +227,11 @@ public function isOutdated() : bool
         return $this->page()['header']['outdated'] ?? false;
     }
 
+    public function needsCleanup() : bool
+    {
+        return $this->page()['header']['needs_cleanup'] ?? false;
```

Comment:
```suggestion
        return $this->page['header']['needs_cleanup'] ?? false;
```

Reply:
`page` is only a function here isn't it

Reply:
`page()` has been removed, it's `page` now